### PR TITLE
[Documentation] Concepts: Taxation

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes
| New docs?    | no
| BC breaks?      | no
| Related tickets | partially #5275 
| License         | MIT

### CODE REVIEW

File: docs/book/taxation.rst [hunk 1]
```diff
@@ -30,27 +43,82 @@ A tax rate is essentially a percentage amount charged based on the sales price.
...
 
+TaxRates includedInPrice
+''''''''''''''''''''''''
+
+The **TaxRate** entity has a field for configuring if you would like to have taxes included in the price of a subject or not.
+
+If you have a TaxCategory with a 10% TaxRate *includedInPrice*, then the price shown on the ProductVariant in that
+TaxCategory will be increased by 10% all the time.
+
+If the TaxRate *will not be included* the the price of ProductVariant will be shown without taxes, but when this
```
Comment (tuka217) [inline docs/book/taxation.rst]: `If the TaxRate will not be included  **then** the (...)`

File: docs/book/taxation.rst [hunk 2]
```diff
@@ -4,7 +4,7 @@
 Taxation
 ========
 
-Sylius has a very flexible taxation system, which allows you to apply appropriate taxes for different items, billing zones and use custom calculators.
+Sylius has a taxation system, which allows you to apply appropriate taxes for different items, billing zones and using custom calculators.
```
Comment (NeverResponse) [inline docs/book/taxation.rst]: WDYT 'bout `Sylius' taxation system allows you to...` and so on?

File: docs/book/taxation.rst [hunk 3]
```diff
@@ -30,27 +43,82 @@ A tax rate is essentially a percentage amount charged based on the sales price.
 * The tax category that a product must belong to in order to be considered taxable
 * Calculator to use for computing the tax
 
+TaxRates includedInPrice
```
Comment (NeverResponse) [inline docs/book/taxation.rst]: `Taxes included in price`? It's a title, it doesn't need to be specific.

**Comment by pjedrzejewski** [comment]
@TheMadeleine We should add one last thing: setting TaxCategory on ProductVariant. :)

**Comment by pjedrzejewski** [comment]
Thank you Magda! :)

**Comment by Mrkisha** [comment]
Hi,

I have used TaxationBundle outside of Sylius. Would like to contribute to documentation as the existing docs are very hard to follow. This of course goes for using this bundle outside Sylius. I've never contributed to Open Source Software and I would need a little help as well.

**Comment by TheMadeleine** [comment]
Hi @Mrkisha ! Really happy to hear that! :) What help do you exactly need? 

I would suggest reading [this guide](http://docs.sylius.org/en/latest/contributing/documentation/overview.html) first :)

**Comment by Mrkisha** [comment]
@TheMadeleine Ok, so I made a pull request. Was confused how to go with docs, should I have changed the existing or should I have added another page to it and write how I implemented the bundle. So, I went with later. #6441 All comments are welcome.
